commons/tool/src/webapp/js/commons.js

```diff
@@ -124,6 +124,22 @@ commons.switchState = function (state, arg) {
                     editorCancelButton.prop('disabled', false);
                 }
                 editor.focus();
+            }).on('drop', function (e) {
+                // clear placeholder text
+                if (this.innerHTML == commons.i18n['post_editor_initial_text']) {
+                    this.innerHTML = '';
+                    $('#commons-editor-post-button').prop('disabled', false);
+                    editorPostButton.prop('disabled', false);
+                    editorCancelButton.prop('disabled', false);
+                }
+                editor.focus();
+
+                // get data
+                var dt = e.originalEvent.dataTransfer;
+                if (!dt) dt = window.dataTransfer;
+                var dropped = dt.getData('text');
```

Comment:
```suggestion
                const dropped = dt.getData('text');
```

Reply:
Happy to make this change, as well as the suggestion above, but I copied it from the existing "paste" just below at line 131/147.  I can change those too if desired.